controllers/articles.controller.js

```diff
@@ -40,4 +41,14 @@ exports.postCommentForArticle = (request, response, next) => {
      response.status(201).send({comment: comment});
    })
    .catch(next);
-};
\ No newline at end of file
+};
+
+exports.patchArticleByArticleId = (request, response, next) => {
+  const articleId = request.params.article_id;
+  const { inc_votes } = request.body;
+  updateArticleByArticleId(articleId, inc_votes)
+  .then((article) => {
+    response.status(200).send({article : article});
```

Comment:
This can be reduced to ```{article}``` to send back an object with the key of article as you're using article to represent data as well.